tests/integration/ci-runner.py

```diff
@@ -825,7 +810,7 @@ def run_flaky_check(self, repo_path, build_path, should_fail=False):
                 result_state = "failure"
                 if not should_fail:
                     break
-            assert len(counters["FLAKY"]) == 0 or should_fail
+            assert should_fail
```

Comment:
this asserts breaks `Integration tests flaky check`, because `should_fail` is set to `False`.
I think we simply need to return this